ypratham/invoice-generator
# (Add) Spotify-Inspired Theme for Invoice Generator

## Description

This PR introduces a new theme inspired by the Spotify UI to the invoice generator. The goal is to create a visually appealing and modern theme for digital invoices, especially for users who prefer a soft copy. While most invoices are typically uncolored, this new theme will provide a fresh and engaging way to display information, making it ideal for online use.

### Key Features:

- **Spotify-inspired color palette**: A dark theme with green accents that resemble Spotify's aesthetic.
- **Modern typography**: Clean and legible fonts that match the modern look of the theme.

## Screenshots
![image](https://github.com/user-attachments/assets/ed2605c7-469c-4019-8483-34c48fd7c393)


## How to Test

1. Pull the branch locally.
2. Apply the theme to an invoice in the generator.
3. View the invoice on both desktop and mobile devices to ensure responsiveness and appearance.

## Checklist

- [X] Dark theme with green accents implemented.
- [X] Typography matches the theme.

## Related Issues

#16 

---
Let me know if any changes are needed, please add `hacktoberfest-accepted` tag to PR after merging.
